python/4_lens.i

```diff
@@ -33,25 +33,22 @@
 %ignore crpropa::Pixelization::nPix( uint8_t order );
 
 %apply double &INOUT {double &phi, double &theta};
-%ignore MagneticLens::transformModelVector(double *,double) const;
+%ignore MagneticLens::transformModelVector(double *, double) const;
 %include "crpropa/magneticLens/MagneticLens.h"
-%template(LenspartVector) std::vector< crpropa::LensPart *>;
+%template(LenspartVector) std::vector<crpropa::LensPart*>;
 
-#ifdef WITHNUMPY
-%extend crpropa::MagneticLens{
-  PyObject * transformModelVector_numpyArray(PyObject *input, double rigidity)
-  {
+
+%extend crpropa::MagneticLens {
+  PyObject * transformModelVector_numpyArray(PyObject *input, double rigidity) {
     PyArrayObject *arr = NULL;
     PyArray_Descr *dtype = NULL;
     int ndim = 0;
     npy_intp dims[NPY_MAXDIMS];
-    if (PyArray_GetArrayParamsFromObject(input, NULL, 1, &dtype, &ndim, dims, &arr, NULL) < 0)
-    {
+    if (PyArray_GetArrayParamsFromObject(input, NULL, 1, &dtype, &ndim, dims, &arr, NULL) < 0) {
```

Comment:
Something goes wrong on this line. See issue #448 . Please, check behavior of this code on NumPy version 1.21.6.